ambry-api/src/main/java/com/github/ambry/account/Account.java

```diff
@@ -284,6 +284,13 @@ public Container getContainerByName(String containerName) {
     return Collections.unmodifiableCollection(containerIdToContainerMap.values());
   }
 
+  /**
+   * @return the number of containers in this account.
+   */
+  public int getContainerCount() {
+    return containerIdToContainerMap.size();
+  }
+
```

Comment:
Not related to this change but it seems `HELIX_ACCOUNT_SERVICE_ACCOUNT_NAME` in this file can be removed.

Reply:
It could be used by LI code, but I guess we'll find out when we integrate.